src/Apps/IdentityVerification/routes.tsx

```diff
@@ -0,0 +1,18 @@
+import loadable from "@loadable/component"
+import { RouteConfig } from "found"
+import { graphql } from "react-relay"
+
+export const routes: RouteConfig[] = [
+  {
+    path: "/identity-verification/:id",
+    getComponent: () => loadable(() => import("./IdentityVerificationApp")),
```

Comment:
Sorry for all the pattern changes! One small change that was deployed on Friday is setting up a mechanism for preloading the split bundle. See [here](https://github.com/artsy/reaction/blob/master/src/Apps/Artwork/routes.tsx#L9-L12)

Reply:
cc @erikdstock on this. @damassi Can you provide some more context here? What is the desired change and why the shift in pattern?

Reply:
See https://github.com/artsy/reaction/pull/3207